### PR TITLE
Update badge with main

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

File: .ci/jobs/opbeans-dotnet-mbp.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     script-path: .ci/Jenkinsfile
     scm:
     - github:
-        branch-discovery: all
+        branch-discovery: no-pr
```
Comment (cachedout) [inline .ci/jobs/opbeans-dotnet-mbp.yml:13]: this change seems unrelated

File: .ci/jobs/opbeans-dotnet-mbp.yml [hunk 2]
```diff
@@ -42,5 +42,4 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1d
```
Comment (cachedout) [inline .ci/jobs/opbeans-dotnet-mbp.yml:45]: also seems unrelated

File: .ci/jobs/opbeans-dotnet-mbp.yml (hunk 1 above)
Comment (v1v) [reply .ci/jobs/opbeans-dotnet-mbp.yml:13]:
```suggestion
        branch-discovery: all
```

File: .ci/jobs/opbeans-dotnet-mbp.yml [hunk 3]
```diff
@@ -42,5 +42,4 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
```
Comment (v1v) [inline .ci/jobs/opbeans-dotnet-mbp.yml:44]:
```suggestion
        wipe-workspace: 'True'
    periodic-folder-trigger: 1w
```

File: .ci/jobs/opbeans-dotnet-mbp.yml [hunk 4]
```diff
@@ -42,5 +42,5 @@
...
-    periodic-folder-trigger: 1d
+    periodic-folder-trigger: 1w
```
Comment (v1v) [inline .ci/jobs/opbeans-dotnet-mbp.yml:45]:
```suggestion
    periodic-folder-trigger: 1d
```

**Review by cachedout — APPROVED** [review]
